invisible_cities/io/mcinfo_io_test.py

```diff
@@ -104,6 +106,24 @@ def test_is_oldformat_file(ICDATADIR):
     assert not is_oldformat_file(file_in_new)
 
 
+def test_copy_nexus_eventmap(config_tmpdir, ICDATADIR):
+    file_in  = os.path.join(ICDATADIR                                  ,
+                            'nexus_new_kr83m_full.newformat.buffers.h5')
+    file_out = os.path.join(ICDATADIR, 'evtMapCopy.h5')
+
+    expected_evts = np.array([[0, 0], [13, 1]])
+    with tb.open_file(file_out, 'w') as h5out:
+        nexus_evts = copy_nexus_eventmap(h5out, expected_evts[:, 0], [file_in])
+
+        assert len(nexus_evts) == len(expected_evts)
+        assert np.all(nexus_evts == expected_evts[:, 1])
+
+    with tb.open_file(file_in) as h5orig, tb.open_file(file_out) as h5out:
+        assert hasattr(h5out.root.Run, 'eventMap')
+        assert_tables_equality(h5out .root.Run.eventMap,
+                               h5orig.root.Run.eventMap)
+
+
```

Comment:
Can you add a test tho show that the map is not copied and return event are correct in case the input file does not contain the table?